core/drand_beacon_control.go

```diff
@@ -1171,7 +1171,7 @@ func (bp *BeaconProcess) StartFollowChain(req *drand.StartSyncRequest, stream dr
 	}
 
 	// register callback to notify client of progress
-	cbStore := beacon.NewCallbackStore(ss)
+	cbStore := beacon.NewCallbackStore(bp.log, ss)
```

Comment:
This is not ideal, because the `bp.log` logger will change during every resharing, but I doubt we have a way of propagating it properly to the CallbackStore...